.github/workflows/release.yml

```diff
@@ -12,7 +12,7 @@ on:
       - '**.md'
   create:
     tags:
-      - 'v[0-9].[0-9].[0-9]+*'
+      - 'v[0-9].[0-9].[0-9]+.*'
```

Comment:
This isn't correct either - this translates to e.g. 'v1.1.1.*'.
We just need to remove the 'create' and just include tags in with the 'push' block (see https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-including-branches-and-tags) 
```
on:
  push:
    branches:
      - release-1.11
    paths-ignore:
      - 'docs/**'
      - 'docs-web/**'
      - 'examples/**'
      - 'examples-of-custom-resources/**'
      - '**.md'
    tags:
      - 'v[0-9].[0-9].[0-9]+*'

Reply:
Actually, this won't work now for our next release - if I'm regexing correctly, each `[0-9]` will match precisely one digit so it should probably be 'v[0-9].[0-9][0-9].[0-9]+*' or something, but I'll verify that.